tests/Metalama.Framework.Tests.AspectTests/Tests/Aspects/CSharp13/PartialIndexers_WithoutImplementation.t.cs

```diff
@@ -0,0 +1,28 @@
+// Final Compilation.Emit failed.
+// Error CS0161 on `get`: `'Target.this[int].get': not all code paths return a value`
+// Error CS0161 on `get`: `'Target.this[long].get': not all code paths return a value`
+partial class Target
+{
+  [TheAspect]
+  partial int this[int i] { get; set; }
+  partial int this[int i]
+  {
+    get
```

Comment:
Is the missing return intentional?

Reply:
It is not. It's the same thing as with partial methods, as we discussed this week. Metalama can implement partial properties, but if they are also implemented by a source generator, there is going to be a conflict. The challenge is rather how to report this conflict.